packages/quantic/scripts/npm/utils.js

```diff
@@ -0,0 +1,36 @@
+const pathlib = require('path');
+
+function quanticIsDependency() {
```

Comment:
😕 
Why not
```js
try {
  require.resolve('@coveo/quantic');
  return true;
} catch (e) {
  return false;
}
```

Reply:
You can then append the rest of the path to it for example `@coveo/quantic/scripts/utils.js`

This should work.
However I'd say, require the sfdx-project.json. it's location is more guaranteed because it is imposed by salesforce

Reply:
Yes,  using ` require.resolve('@coveo/quantic/scripts/npm/utils.js')` should indeed achieve what we want 👌
The  sfdx-project.json file is not included in the package so it will not be present under @coveo/quantic thus I don't think it will do the job which is to figure out if quantic is installed as a dependency or not.